test/integration/buildclient_test.go

```diff
@@ -233,5 +233,5 @@ func (t *testBuildOpenshift) Close() {
 	defer t.lock.Unlock()
 	close(t.stop)
 	t.server.CloseClientConnections()
-	t.server.Close()
+	defer t.server.Close()
```

Comment:
I must have misread the output from Travis, it still the #731 issue. I'm removing that `defer` from there.